conf/app.routes

```diff
@@ -1,7 +1,7 @@
 # microservice specific routes
 
 POST        /v2/declarations                                 uk.gov.hmrc.exports.controllers.DeclarationController.create()
-GET         /v2/declarations                                 uk.gov.hmrc.exports.controllers.DeclarationController.findAll(status: Option[String] ?= None, page: uk.gov.hmrc.exports.models.Page ?= uk.gov.hmrc.exports.models.Page())
+GET         /v2/declarations                                 uk.gov.hmrc.exports.controllers.DeclarationController.findAll(status: Option[String] ?= None, page: uk.gov.hmrc.exports.models.Page ?= uk.gov.hmrc.exports.models.Page(), sortBy: Option[String] ?= None, sortDirection: Option[String] ?= None)
```

Comment:
Would be nicer if we could bind the `DeclarationSort` class here rather than two params

Reply:
I noticed how that was done with the 'Page'.  Good suggestion.

Reply:
Done